src/adapter/src/coord/sequencer.rs

```diff
@@ -105,6 +108,19 @@ impl<S: Append + 'static> Coordinator<S> {
         depends_on: Vec<GlobalId>,
     ) {
         event!(Level::TRACE, plan = format!("{:?}", plan));
+        if session.vars().emit_tracing_notice() {
+            let span_context = tracing::Span::current()
+                .context()
+                .span()
+                .span_context()
+                .clone();
```

Comment:
you shouldn't need this `clone`! if rust complained about temporaries, you can do:
```
            let cur_span = tracing::Span::current();
            let span_context = cur_span
                .context()
                .span()
                .span_context();
```

Reply:
also, til about `is_valid`, seems nice!

Reply:
Yeah I was getting the error on temporaries.
I had to do 
```
let cur_span = tracing::Span::current();
let curr_context_span = ur_span.context().span().span_context();
```
for rust to stop complaining.
Happy to change it to that.

Reply:
@chaas i thought it would be easy, just leave the clone!